redis/src/cluster_client.rs

```diff
@@ -104,6 +104,12 @@ impl ClusterClientBuilder {
         self
     }
 
+    /// Use `build()`.
```

Comment:
This should be in the same place as the previous `open()` method so the diff is more clear.

Reply:
I've deliberately moved deprecated methods to the bottom of the struct to keep them out of the way.

Reply:
I'd prefer that that is done in a separate commit to faciliate review.